Dockerfile

```
@@ -1,11 +1,4 @@
-# The following image is a mirror of CI image, performed as:
-# * Copy the oc login command from https://api.ci.openshift.org/oauth/token/request
-# * oc login api.ci.openshift.org ... # command from the web page
-# * oc registry login
-# * skopeo copy docker://registry.svc.ci.openshift.org/ocp/builder:golang-1.15 docker:/quay.io/redhat-developer/servicebinding-operator:builder-golang-1.15
-FROM quay.io/redhat-developer/servicebinding-operator:builder-golang-1.16 AS builder
-
-USER root
+FROM golang:1.17 as builder
```

@baijum One of the reasons to use the base image coming from quay.io (instead of docker.io) was to avoid the Docker Hub's rate limit. So I created the `quay.io/redhat-developer/go-toolset` repo as you suggested and mirrored the `golang:1.17` image there as `quay.io/redhat-developer/go-toolset:1.17`
```suggestion
FROM quay.io/redhat-developer/go-toolset:builder-golang-1.17 as builder
```